config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
-  devise_for :users
-  root to:"home#index"
-  devise_for :models
+  devise_for :users,
+#  root to:"home#index",
+    #path: 'users',
+    path_names: {
```

Comment:
indent this correctly

Reply:
done, I deleted one comment and indent the rest.